forms-flow-web/src/apiManager/services/metricsServices.js

```diff
@@ -30,8 +30,9 @@ export const fetchMetricsSubmissionCount = (
     /*eslint max-len: ["error", { "code": 170 }]*/
     let url = `${API.METRICS_SUBMISSIONS}?from=${fromDate}&to=${toDate}&orderBy=${searchBy}&pageNo=${pageNo}&limit=${limit}&sortBy=${sortsBy}&sortOrder=${sortOrder}`;
     if (formName) {
-      url += `&formName=${formName}`;
+      url += `&formName=${encodeURIComponent(formName)}`;
```

Comment:
please ensure this function will with all browser

Reply:
Encodeuri component would supprted in most browsers I think..
https://caniuse.com/mdn-javascript_builtins_encodeuricomponent